test/val/val_limits_test.cpp

```diff
@@ -320,3 +321,71 @@ TEST_F(ValidateLimits, StructNestingDepthBad) {
       HasSubstr(
           "Structure Nesting Depth may not be larger than 255. Found 256."));
 }
+
+// clang-format off
```

Comment:
The clang formatting pushed every "<<" of the ostringstream to a new line which made it difficult to see the spir-v text I was trying to build. 

Keeping clang-format off allows us to see what string goes on each line of the spir-v code. I thought it might be more readable this way. The entire function is just building the spir-v code string.